main.js

```diff
@@ -119,19 +119,12 @@ function startApp() {
   }
 
   function handleUnhandledException(err) {
-    bt.report(err, {}, handleFinishedReport);
+    Sentry.captureException(err);
```

Comment:
We should at the very least send along the version number.  Does sentry's node library automatically pull that from the `package.json` or do we need to send it along?

Reply:
I believe it should be added manually, this is a sample report from these modifications https://sentry.io/organizations/streamlabs-obs/issues/966567397/events/51437036986/?project=251674&statsPeriod=14d&utc=false

Reply:
Looking into this

Reply:
I was unable to test but I believe my modifications should populate the version and release tags (Sentry is probably filtering me out)

Reply:
Since I don't know what is accessible from the `main.js` file I wasn't sure what extra info to send, but it should be easy to add if necessary. Just ping me with what need to be sent.